src/components/breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "@testing-library/jest-dom";
```

Comment:
Don't think this import is needed. I think `setupTests.ts` should be doing this for every test. If it isn't working for you then perhaps we need to tweak the config.

Reply:
Removed it and all still seems to work alright